TADashboard/src/main/java/utils/common/Constants.java

```diff
@@ -3,14 +3,36 @@
 import utils.helper.PropertiesHelper;
 
 public class Constants {
+    public static final int SHORT_TIME_WAIT = 1;
+    public static final int LONG_TIME_WAIT = 10;
 
-	// Common Constants
-	public static final String BROWSER_SETTING_FILE = "src/test/resources/browsers.setting.ini";
-	public static final int TIME_OUT = Integer
-			.parseInt(PropertiesHelper.getPropValue("driver.timeout"));
-	public static final int SHORT_TIME_OUT = Integer
-			.parseInt(PropertiesHelper.getPropValue("driver.shortTimeout"));
+    // Common Constants
+    public static final String BROWSER_SETTING_FILE = "src/test/resources/browsers.setting.ini";
+    public static final int TIME_OUT = Integer
+            .parseInt(PropertiesHelper.getPropValue("driver.timeout"));
+    public static final int SHORT_TIME_OUT = Integer
+            .parseInt(PropertiesHelper.getPropValue("driver.shortTimeout"));
 
-	// AUT Constants
-	public static final String AUT_URL = PropertiesHelper.getPropValue("aut.url");
-}
+    // AUT Constants
+    public static final String AUT_URL = PropertiesHelper.getPropValue("aut.url");
+
+    //Pre-set Data Profiles
+    public static final String[] PRE_SET_DATA_PROFILES = {
```

Comment:
what is this?

Reply:
This is the whole Pre-Set Data Profile, which I need to capture using xpath. And this has a special non-printable character nbsp (Code 0160).

Reply:
why do you need a constant for those?

Reply:
I have fixed it. Please check again